src/components/Button/Button.tsx

```diff
@@ -8,17 +8,23 @@ type ButtonProps = {
   onClick?: () => void;
   className?: string;
   type?: 'button' | 'submit';
+  variant: 'primary' | 'secondary';
 } & ComponentProps<'button'>;
 
 export const Button: ChildrenFC<ButtonProps> = ({
   children,
   onClick,
   className,
+  variant = 'primary',
   type = 'button',
   ...props
 }) => (
   <button
-    className={clsx(style.button, className)}
+    className={clsx(
+      style.base,
+      variant === 'primary' ? style.primary : style.secondary,
```

Comment:
Use the object syntax
```suggestion
{ [style.primary]: variant === 'primary',
  [style.secondary]: variant === 'secondary' 

```